addons/website_payment/__openerp__.py

```diff
@@ -15,6 +15,7 @@
     'data': [
         'views/website_payment_templates.xml',
         'views/website_settings_payment.xml',
+        'data/transfer.xml',
     ],
-    'auto_install': False,
+    'auto_install': True,
```

Comment:
Seems legit to me to be able to publish a payment method if 'website' and 'payment' are published without checking a option.